releasenotes/notes/libddwaf-1.0.18-update-61c1513f3e632195.yaml

```
@@ -0,0 +1,3 @@
+---
+fixes:
+  - Fixes build issues on older MacOS versions by updating `libddwaf` to 1.0.18
```

```suggestion
  - Fixes build issues on older MacOS versions by updating ``libddwaf`` to 1.0.18
```

This is RST not Markdown